src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexMatcher.Dgml.cs

```diff
@@ -0,0 +1,212 @@
+            writer.WriteLine("            <Setter Property=\"FontSize\" Value=\"12\" />");
+            writer.WriteLine("            <Setter Property=\"FontFamily\" Value=\"Arial\" />");
+            writer.WriteLine("        </Style>");
+            writer.WriteLine("        <Style TargetType=\"Link\" GroupLabel=\"NonEpsilonTransition\" ValueLabel=\"True\">");
+            writer.WriteLine("            <Condition Expression=\"HasCategory('NonEpsilonTransition')\" />");
+            writer.WriteLine("            <Setter Property=\"FontSize\" Value=\"18\" />");
+            writer.WriteLine("            <Setter Property=\"FontFamily\" Value=\"Arial\" />");
+            writer.WriteLine("        </Style>");
+            writer.WriteLine("        <Style TargetType=\"Link\" GroupLabel=\"StartTransition\" ValueLabel=\"True\">");
+            writer.WriteLine("            <Condition Expression=\"HasCategory('StartTransition')\" />");
+            writer.WriteLine("        </Style>");
+            writer.WriteLine("        <Style TargetType=\"Link\" GroupLabel=\"EpsilonTransition\" ValueLabel=\"True\">");
+            writer.WriteLine("            <Condition Expression=\"HasCategory('EpsilonTransition')\" />");
+            writer.WriteLine("            <Setter Property=\"StrokeDashArray\" Value=\"8 8\" />");
+            writer.WriteLine("        </Style>");
+            writer.WriteLine("        <Style TargetType=\"Link\" GroupLabel=\"FinalLabel\" ValueLabel=\"False\">");
+            writer.WriteLine("            <Condition Expression=\"HasCategory('FinalLabel')\" />");
+            writer.WriteLine("            <Setter Property=\"StrokeDashArray\" Value=\"8 8\" />");
+            writer.WriteLine("        </Style>");
+            writer.WriteLine("        <Style TargetType=\"Node\" GroupLabel=\"StateInfo\" ValueLabel=\"True\">");
+            writer.WriteLine("            <Setter Property=\"Stroke\" Value=\"white\" />");
+            writer.WriteLine("            <Setter Property=\"FontSize\" Value=\"18\" />");
+            writer.WriteLine("            <Setter Property=\"FontFamily\" Value=\"Arial\" />");
+            writer.WriteLine("        </Style>");
+            writer.WriteLine("        <Style TargetType=\"Node\" GroupLabel=\"DFAInfo\" ValueLabel=\"True\">");
+            writer.WriteLine("            <Setter Property=\"Stroke\" Value=\"white\" />");
+            writer.WriteLine("            <Setter Property=\"FontSize\" Value=\"18\" />");
+            writer.WriteLine("            <Setter Property=\"FontFamily\" Value=\"Arial\" />");
+            writer.WriteLine("        </Style>");
+            writer.WriteLine("    </Styles>");
+            writer.WriteLine("</DirectedGraph>");
+
+            // This function gathers all transitions in the given builder and groups them by (source,destination) state ID.
+            // It also counts the number of transitions found.
```

Comment:
> It also counts the number of transitions found.

What is this referring to?

Reply:
Ah, the comment's fallen out of sync during development. Thanks!